entry.go

```diff
@@ -124,7 +124,9 @@ func (entry Entry) log(level Level, msg string) {
 
 func (entry *Entry) Debug(args ...interface{}) *Entry {
 	if entry.Logger.Level >= DebugLevel {
-		entry.log(DebugLevel, fmt.Sprint(args...))
+		hooksDone := make(chan struct{}, 1)
+		entry.HooksDone = hooksDone
```

Comment:
This is racy, multiple goroutines can invoke a `Debug/Info/Warn/...` function on the `*Entry` at the same time.